internal/engine/sqlite/stdlib.go

```diff
@@ -5,18 +5,29 @@ import (
 	"github.com/kyleconroy/sqlc/internal/sql/catalog"
 )
 
-// TODO: fill out sqlite functions from:
+// sqlite functions from:
 // 		 https://www.sqlite.org/lang_aggfunc.html
 // 		 https://www.sqlite.org/lang_mathfunc.html
 //		 https://www.sqlite.org/lang_corefunc.html
 
 func defaultSchema(name string) *catalog.Schema {
 	s := &catalog.Schema{Name: name}
 	s.Funcs = []*catalog.Function{
+		// Aggregation Functions
+		{
+			Name: "AVG",
+			Args: []*catalog.Argument{
+				{
+					Type: &ast.TypeName{Name: "any"},
+				},
+			},
+			ReturnType:         &ast.TypeName{Name: "real"},
+			ReturnTypeNullable: true,
+		},
 		{
 			Name:       "COUNT",
 			Args:       []*catalog.Argument{},
-			ReturnType: &ast.TypeName{Name: "bigint"},
+			ReturnType: &ast.TypeName{Name: "integer"},
```

Comment:
This will change quite a bit of generated code. Is it documented that SQLite will always return an int here?

Reply:
In SQLite world, `bigint` and `integer` is totally same.

https://www.sqlite.org/datatype3.html